less_users/tests/test_check_if_active.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from django.urls import reverse
+from freezegun import freeze_time
+
+from datetime import date, timedelta
+
+from less_users.models import UserChallenge
+
+
+def test_check_if_active_before_duration_time(user, challenge_3_month):
+    with freeze_time(date.today() - timedelta(days=challenge_3_month.duration - 1)):
+        userchallenge = UserChallenge.objects.create(
+            user=user, challenge=challenge_3_month
+        )
+
+    userchallenge.check_if_active
+
+    assert userchallenge.is_active
+
+
+def test_check_if_active_after_duration_time(user, challenge_3_month):
+    with freeze_time(date.today() - timedelta(days=challenge_3_month.duration + 1)):
+        userchallenge = UserChallenge.objects.create(
+            user=user, challenge=challenge_3_month
+        )
+
+    userchallenge.check_if_active
+
+    assert not userchallenge.is_active
+
+
+def test_check_if_not_active_when_deactivated(user, challenge_3_month):
+    userchallenge = UserChallenge.objects.create(user=user, challenge=challenge_3_month)
+    userchallenge.is_active = False
+    userchallenge.check_if_active
+
+    assert not userchallenge.is_active
+
+
+@pytest.mark.skip
```

Comment:
Po pierwsze test jest skipnięty, po drugie to nie jest do `check_if_active` tylko do event stop

Reply:
przerzuciłam go do innej grupy testów